src/MessagingInput/MessagingInput.tsx

```diff
@@ -60,36 +63,43 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
 
   return (
     <FlexBox className={cx(cssClass("Container"), className)} column alignItems={ItemAlign.START}>
-      <label htmlFor={TEXT_FIELD_NAME} className={cssClass("TextFieldLabel")}>
+      <label
+        htmlFor={TEXT_FIELD_NAME}
+        className={cssClass(replyTo ? "TextFieldLabelHidden" : "TextFieldLabel")}
```

Comment:
using class for hiding so the label is still read by a screenreader if using a placeholder instead